addons/hr/views/hr_employee_views.xml

```diff
@@ -62,7 +62,7 @@
                         </div>
                         <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
                         <field name="avatar_128" invisible="1"/>
-                        <div class="row justify-content-between position-relative w-100 m-0 mb-2">
+                        <div class="position-relative w-100 justify-content-between row m-0 mb-2">
```

Comment:
There's no added value here, in fact the order you changed is not right 😬 
`row` should be first (layout class) and `w-100` should be before `m-0`.
But since there's no new/removed class, you can keep it as before.